src/quo2/components/colors/color_picker/view.cljs

```diff
@@ -32,20 +36,24 @@
            on-press
            blur?]}]
   (let [border? (and (not blur?) (and secondary-color (not selected?)))]
-    [rn/touchable-opacity
-     {:style               (style/color-button color selected?)
-      :accessibility-label :color-picker-item
-      :on-press            #(on-press name)}
-     [rn/view
-      {:style (style/color-circle color border?)}
-      (when (and secondary-color (not selected?))
-        [rn/view
-         {:style (style/secondary-overlay secondary-color border?)}])
-      (when selected?
-        [icon/icon :i/check
-         {:size  20
-          :color (or secondary-color
-                     colors/white)}])]]))
+    (if-not name
```

Comment:
I think the check is not correct. `name` will always have a value. Changing it to `color` will fix this issue:https://github.com/status-im/status-mobile/pull/16291#pullrequestreview-1485715685.

Reply:
Yep, thanks @J-Son89! :)